### PR TITLE
Update documentation for Microsoft.VisualBasic.Forms for release 6.0

### PR DESCRIPTION
**Pull request by dreddy-work**
## Summary

Documentation for VisualBasic.Forms were missing due to a glitch in the `DocPortingTool `from the earlier check-ins. This PR is generated after updating `DocPortingTool `and ran against the latest source in .NET 6.0.

### CODE REVIEW

**Comment by dreddy-work** [comment]
cc: /@KlausLoeffelmann

**Comment by KlausLoeffelmann** [comment]
Also @KathleenDollard for FYI!

**Comment by dreddy-work** [comment]
Everything LGTM.

**Comment by dreddy-work** [comment]
@carlossanlop I can not add reviewers. Please help get this available to 6.0.

File: xml/Microsoft.VisualBasic.ApplicationServices/ApplyApplicationDefaultsEventHandler.xml [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+    <Parameter Name="sender" Type="System.Object" />
+    <Parameter Name="e" Type="Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs" />
+  </Parameters>
+  <ReturnValue>
+    <ReturnType>System.Void</ReturnType>
+  </ReturnValue>
+  <Docs>
+    <param name="sender">To be added.</param>
+    <param name="e">To be added.</param>
+    <summary>Signature for the ApplyApplicationDefaults event handler.</summary>
```
Comment (gewarren) [inline xml/Microsoft.VisualBasic.ApplicationServices/ApplyApplicationDefaultsEventHandler.xml:31]:
```suggestion
    <summary>Represents the callback method that will handle the <see cref="E:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.ApplyApplicationDefaults" /> event.</summary>
```

File: xml/Microsoft.VisualBasic.ApplicationServices/ConsoleApplicationBase.xml [hunk 2]
```diff
@@ -1,348 +1,358 @@
...
+      </Attributes>
+      <Parameters />
+      <Docs>
+        <summary>Initializes a new instance of the <see cref="T:Microsoft.VisualBasic.ApplicationServices.ConsoleApplicationBase" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+We have to have a parameterless constructor because the platform specific Application
```
Comment (gewarren) [inline xml/Microsoft.VisualBasic.ApplicationServices/ConsoleApplicationBase.xml:91]: I'm not sure these remarks are relevant to the user?

File: xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml [hunk 3]
```diff
@@ -1,1796 +1,1806 @@
...
+      </Attributes>
+      <Parameters />
+      <Docs>
+        <summary>Initializes a new instance of the <see cref="T:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+We have to have a parameterless ctor because the platform specific Application object
```
Comment (gewarren) [inline xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml:94]: Relevant?

File: xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml [hunk 4]
```diff
@@ -1,1796 +1,1806 @@
...
+        <Attribute FrameworkAlternate="net-6.0">
+          <AttributeName Language="C#">[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.HighDpiMode</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets or sets the HighDpiMode for the Application.</summary>
```
Comment (gewarren) [inline xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml:418]:
```suggestion
        <summary>Gets or sets the HighDpiMode for the application.</summary>

```

File: xml/Microsoft.VisualBasic.Logging/Log.xml [hunk 5]
```diff
@@ -1,872 +1,880 @@
...
+      </Attributes>
+      <Parameters />
+      <Docs>
+        <summary>Initializes a new instance of the <see cref="T:Microsoft.VisualBasic.Logging.Log" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Right now we only support WinApp as an application platform
```
Comment (gewarren) [inline xml/Microsoft.VisualBasic.Logging/Log.xml:93]: Is this relevant to the user?

**Comment by dreddy-work** [comment]
@KlausLoeffelmann , Can you please go over these suggestions and respond as necessary.

**Comment by gewarren** [comment]
Close/reopen for fresh build.

**Comment by carlossanlop** [comment]
@dreddy-work @KlausLoeffelmann ping. We don't have much time to generate an updated intellisense package that we can include in the 6.0 Release. There are several comments that need to be addressed.

**Comment by carlossanlop** [comment]
@dreddy-work can you please resolve the merge conflicts so we can merge this?

I deleted my previous comment, apparently the GitHub tab in my browser did not get refreshed and I just noticed you already addressed all the comments.

**Comment by dreddy-work** [comment]
> @dreddy-work can you please resolve the merge conflicts so we can merge this? Can you please also apply the last suggestions? We have little time to get this merged in 6.0.

@KlausLoeffelmann , for those last suggestions, I was looking to get confirmation from you.
